aten/src/ATen/test/vec_test_all_types.h

```diff
@@ -1090,6 +1090,38 @@ void test_binary(
     }
 }
 
+template<typename Op1, typename Op2>
+void test_binary_fp8_e4m3(
+    Op1 ScalarFunction,
+    Op2 VecFunction,
+    bool is_bit_wise = false) {
+    #if defined(CPU_CAPABILITY_AVX512) && !defined(__APPLE__) && !defined(_MSC_VER)
+    for (const auto i : c10::irange(10)) {
```

Comment:
the test case won't touch denorm values or verify NaN behavior.

suggest that iterate from [0, 255) bit values and do the tests, which will cover all the possible values for fp8.

Reply:
Thanks for the comment. For the UT: `./bin/vec_test_all_types_AVX512 --gtest_filter=FP8E4M3Test.FP8E4M3ConversionFloat` to test convert from/to fp32, iterate from [0, 255) bit values to covert all the possible values.